Text/Internal/TemplateUtils.hs

```diff
@@ -0,0 +1,20 @@
+{-# LANGUAGE TemplateHaskell #-}
+module Text.Internal.TemplateUtils  where
```

Comment:
It looks like all use sites are using this function and then reading the file. How about adding a function here that both adds the dependent file and returns the file contents?

Reply:
looks like we can use `readFileQ` to do just that,